src/neuroconv/tools/audio/audio.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional
+from warnings import warn
+
+from hdmf.backends.hdf5 import H5DataIO
+from ndx_sound import AcousticWaveformSeries
+from pynwb import NWBFile
+
+from neuroconv.tools.hdmf import SliceableDataChunkIterator
+from neuroconv.utils import ArrayType
+
+
+def add_acoustic_waveform_series(
+    acoustic_series: ArrayType,
+    rate: float,
+    nwbfile: NWBFile,
```

Comment:
```suggestion
    acoustic_series: ArrayType,
    nwbfile: NWBFile,
    rate: float,
```

Argument structure for tools is generally `neurodata_type_to_add` (which is fine here) and then `nwbfile` (which is modified in-place)